length_indicator.module

```diff
@@ -65,6 +65,13 @@ function length_indicator_field_widget_third_party_settings_form(WidgetInterface
 
     $indicator_opt = $plugin->getThirdPartySetting('length_indicator', 'indicator_opt');
 
+    $element['indicator_opt']['unit'] = [
+      '#type' => 'radios',
+      '#title' => t('Unit for length'),
+      '#options' => ['char' => t('Character length (char)'), 'px' => t('Width in pixel (px)')],
+      '#default_value' => isset($indicator_opt['unit']) ? $indicator_opt['unit'] : 'char',
+    ];
+
     $element['indicator_opt']['optimin'] = [
```

Comment:
The configuration options have to change after selecting px/char. For pix we need the length in px and a selection of the font and font size to use.